src/Microsoft.Health.Fhir.CosmosDb.Initialization/Registration/DataPlaneCosmosDbRegistrationExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System;
+using System.Linq;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.DependencyInjection;
+
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.Registration
+{
+    public static class DataPlaneCosmosDbRegistrationExtensions
+    {
+        public static void AddDataPlaneDepencies(this IServiceCollection services)
```

Comment:
Probably instead of calling this method "AddDataPlaneDepencies", we should call it "AddCosmosDbInitializationDependencies"

Reply:
done